src/hooks/usePoolData.ts

```diff
@@ -301,6 +310,39 @@ export default function usePoolData(
           .mul(allocPoint)
           .div(totalAllocPoint)
       }
+
+      const metaSwapAddress = POOL.metaSwapAddresses?.[chainId]
+      let isMigrated = false
+      if (metaSwapAddress) {
+        try {
+          const migrationMapRes = await migratorContract.migrationMap(
+            metaSwapAddress,
+          )
+          isMigrated =
+            migrationMapRes.oldPoolLPTokenAddress !==
+            "0x0000000000000000000000000000000000000000"
+              ? true
+              : false
+        } catch (err) {
+          console.error(err)
+        }
+      }
+      // Edge case for Stable Coin Pool as it has no metaSwapAddress but is still migrated.
```

Comment:
I think this logic makes two assumptions which could be refined:
- that by default only metapools get migrated
- that the stablecoin pool is a special case.

Actually, any swap pool can get migrated. Thus the logic should look like 
```
const poolAddressToCheckMigrationStatus = POOL.metaSwapAddresses?.[chainId] || poolAddress
if (poolAddressToCheckMigrationStatus) {...}
```

Also I'd use `ethers.constants.AddressZero` instead of typing it out yourself.
Also just `isMigrated = migrationMapRes.newPoolAddress !== AddressZero // is already boolean`